Ducky-Chickens/bangazonCLI
# Add Product to Customer's Inventory (Issue Ticket #4 - TM)

# Description
Provides capability in given prompt to insert a new row into product table. 

## Related Ticket(s)
Issue Ticket: #4 

## Problem to Solve
Gives UI capabilities to add new products per active customer


## Steps to Test Adding Customer Products
+ Run `db:reset`
+ Run `npm i -g`
+ Run `bangazon`
+ Select option `2` to activate a customer
+ Select option `4` upon returning to the main prompt to begin adding a product
+ Enter data to the following prompts appropriately
+ CLI should notify insertion of data by stating
```
<product title entered> added to line <primary key of product added>
```

## Testing


**To Run Tests**:
+ Please remove `.skip` method from test function in `addCustomerProduct.test.js` at line `36`
+ Run `npm test`
